examples/multicluster/usecases/floating_nse_composition/cluster1/nse-firewall/config-patch.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nse-firewall-vpp
+spec:
+  template:
+    spec:
+      containers:
+        - name: nse
+          volumeMounts:
+            - mountPath: /etc/vppagent-firewall/config.yaml
```

Comment:
Could you delete "vppagent" prefix from names? Because in fact we haven't used vppagent for a long time